### ISSUE TITLE
Why can't I assert on undefined?

### ISSUE THREAD

**Issue by brianbegy** [webdriverio/expect-webdriverio#1144]
Node 18, typescript 4.9.

I am upgrading my tests that depended on `wdio/sync` (node 14 EOL-ed) and I am unable to use assertions where my `expected` is `undefined.`. 

I have a lot of use cases around making GraphQL requests and returning back an object that looks like `{data?:SomeObject, errors?:GraphQLError[]}`. 

e.g. 
```
const result = await makeCallToApiFunctionThatUsesBrowserToDoSomething();
expect(result.errors).toBeFalsy();
```

Is there any reason the type 

```
type ExpectType = WebdriverIO.Element | Promise<WebdriverIO.Element> | ChainablePromiseElement | ChainablePromiseArray  
```

cannot be 

```
type ExpectType = WebdriverIO.Element | Promise<WebdriverIO.Element> | ChainablePromiseElement | ChainablePromiseArray | unknown | any | undefined

```

**Comment by christian-bromann** [comment]
This is a known bug and we are working on a fix for it.

**Comment by christian-bromann** [comment]
Fixed in `v4.2.2`
